bower.json

```diff
@@ -16,15 +16,16 @@
     "package.json"
   ],
   "dependencies": {
-    "purescript-argonaut-core": "^3.1.1",
-    "purescript-codec": "^2.1.0",
-    "purescript-generics-rep": "^5.4.0",
-    "purescript-variant": "^4.0.0",
-    "purescript-typelevel-prelude": "^2.6.0",
-    "purescript-type-equality": "^2.1.0"
+    "purescript-argonaut-core": "^4.0.0",
+    "purescript-codec": "v3.0.0",
```

Comment:
Humpity bump

Reply:
Bumped that already :P 3.0.0 is the version you just released ;)

Reply:
🤦‍♂️